package.json

```diff
@@ -34,7 +34,8 @@
     "demo:reload": "npm run build && cd demo && npm i file:../",
     "docs": "typedoc",
     "modulereport": "tsc --noEmit --esModuleInterop scripts/moduleReport.ts && esr scripts/moduleReport.ts",
-    "cspell": "cspell '{src,test}/**' './*.md'"
+    "cspell": "cspell '{src,test}/**' './*.md'",
+    "test:no-integration": "vitest run --exclude \"test/**/*.integration.test.ts?(x)\""
```

Comment:
Perhaps flip this on its head to be `test:unit`?

Reply:
One thing we could actually do here, we could define the different suites in `vitest.workspace.ts` and then reference them here

Reply:
Renamed to test:unit and added `test:unit-chat` and `test:unit-react` to be able to run the different projects separately.